src/Config.php

```diff
@@ -33,6 +33,7 @@ public function getDefaultConfig()
             'run_scripts' => 1,
             'security_updates_only' => 0,
             'number_of_concurrent_updates' => 0,
+            'skip_security_updates_on_concurrent_limit' => 1,
```

Comment:
I would prefer to have the option named in the opposite way. Something like `allow_security_updates_on_concurrent_limit`

Since that would then default to 0, it feels more like an opt in type of behavior when named like that

Reply:
Changed is done 👉🏻 https://github.com/violinist-dev/violinist-config/compare/96996d5a18025e7f5af45a57d96070980a0f0d88..dd534c8b96d6aa30017b5e5b02ba66397f5843af